roles/gerrit/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Install Java and python-bugzilla
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - java-1.8.0-openjdk
+  - python-bugzilla
+
+- name: Create a user for review
+  user:
+    name: "{{gerrit_user}}"
+    home: "{{gerrit_user_home}}"
+
+
+- name: Check if Gerrit is installed
+  stat:
+    path: "{{gerrit_user}}/{{gerrit_user_home}}"
+    register: gerrit_status
+
+- name: "Fail when Gerrit is not setup"
+  fail:
+    msg: "Please setup Gerrit at {{gerrit_user}}/{{gerrit_user_home}}"
+  when: gerrit_status.status.islnk is not defined
+
+- name: Setup the replication.config file
+  template:
+    src: replication.config
+    dest: "{{gerrit_user_home}}/{{gerrit_home}}/etc/replication.config"
+    owner: "{{gerrit_user}}"
+    mode: 0755
+
+- name: Create the .ssh folder
+  file:
+    path: "{{gerrit_user_home}}/.ssh"
+    owner: "{{gerrit_user}}"
+    mode: 0655
+    state: directory
+
+- name: Setup the SSH config file
+  template:
+    src: ssh.config
+    dest: "{{gerrit_user_home}}/.ssh/config"
+    owner: "{{gerrit_user}}"
+    mode: 0644
+
+- name: Setup the replication keys
+  copy:
+    src: "{{item.keyfile}}"
+    dest: "{{gerrit_user_home}}.ssh/{{item.keyfile}}"
+    owner: "{{gerrit_user}}"
+    mode: 0700
+  with_items: gerrit_replication
+
+- name: Cron to delete old log files
+  cron:
+    name: "Clean logs"
+    special_time: "weekly"
+    job: "find {{gerrit_user_home}}/{{gerrit_home}}/logs -mtime +15 -delete"
+    user: {{gerrit_user}}
+
+- name: Setup python-bugzilla
+  command: "bugzilla login {{bugzilla_user}} {{bugzilla_pass}}
+  creates: "{{gerrit_user_home}}/.bugzillacookies"
```

Comment:
Creates is special, you need to use that syntax:

https://github.com/OSAS/ansible-role-freeipa/blob/master/tasks/client.yml#L17